subprojects/docs/docs.gradle

```
@@ -402,6 +402,30 @@ samples {
                 from(sampleDirectory.dir("common"))
             }
             // TODO (donat) sanity checks probably don't do anything for samples located one level down in the gradle/kotlin directories
+        }*/
+
+        compositeBuildsBasics {
+            sampleDirectory = file("src/samples/build-organization/compositeBuilds/basic")
+            description = "Defining and using a composite build"
+            category = "Build organization"
+        }
+
+        compositeBuildsDeclaredSubstitutions {
+            sampleDirectory = file("src/samples/build-organization/compositeBuilds/declared-substitution")
+            description = "Declaring the dependencies substituted by an included build"
+            category = "Build organization"
+        }
+
+        compositeBuildsHierarchicalMultiRepo {
+            sampleDirectory = file("src/samples/build-organization/compositeBuilds/hierarchical-multirepo")
+            description = "Using composite builds in a hierarchical multirepo project"
```

Like above, this should describe the use case. Maybe something like
```suggestion
            description = "Defining and using a composite build to combine multiple independent builds."
```